src/reliability.jl

```diff
@@ -43,20 +41,20 @@ julia> cov_X = [10 6 6 6;
                 6 6 12 6;
                 6 6 6 13];
 
-julia> crombach_alpha(cov_X)
+julia> crombachalpha(cov_X)
 Reliability for all items: 0.8136
 
 Reliability if an item is dropped:
-item 1: 0.75
+item 1: 0.7500
 item 2: 0.7606
 item 3: 0.7714
 item 4: 0.7836
 ```
 """
-function crombach_alpha(covmatrix::AbstractMatrix{<:Real})
+function crombachalpha(covmatrix::AbstractMatrix{<:Real})
     isposdef(covmatrix) || throw(ArgumentError("Covariance matrix must be positive definite."))
     k = size(covmatrix, 2)
-    k > 1  || throw(ArgumentError("Covariance matrix must have more than one column."))
+    k >= 1  || throw(ArgumentError("Covariance matrix must have more than one column."))
```

Comment:
Is this change intended? The error message doesn't match the check now.

Reply:
Yes, you are right. I made a terrible confusion. Let's revert:

```suggestion
    k > 1  || throw(ArgumentError("Covariance matrix must have more than one column."))
```